napari/layers/surface/surface.py

```diff
@@ -27,22 +27,48 @@ class Surface(IntensityVisualizationMixin, Layer):
     """
     Surface layer renders meshes onto the canvas.
 
+    Surfaces may be colored by:
+        * setting `vertex_values`, which colors the surface with the selected
+          `colormap` (default is uniform ones)
+        * setting `vertex_colors`, which replaces/overrides any color from
+          `vertex_values`
+        * setting both `texture` and `texcoords`, which applies the texture
+          (which may be transparent) on top of the colors from
+          `vertex_values` or `vertex_colors`
```

Comment:
Texture and vertex color are just additive right? If so, it's not super clear from this (especially the caveat about being transparent). If not, what is it actually doing?

Reply:
@brisvag They can't be additive, or you couldn't see the black eyes of the cow in the versions with vertex values or vertex colors.

Reply:
True... I'm not sure what it's doing then :/

Reply:
Ah I found it in the vispy docs:
> Note the texture is applied by multiplying the texture color by the Visual’s produced color. By specifying color=”white” when creating a MeshVisual the result will be the unaltered texture value. Any other color, including the default, will result in a blending of that color and the color of the texture.

Reply:
Multiplicative blending! Didn't know that was a thing. 😃